vmm/src/acpi.rs

```diff
@@ -28,6 +24,22 @@ struct PCIRangeEntry {
     _reserved: u32,
 }
 
+#[repr(packed)]
+#[derive(Default)]
+struct MemoryAffinity {
```

Comment:
This is a question: what is the best reference to get the 'right' ACPI table format (e.g. the SRAT table format)? The APIC spec from the UEFI website? 

Also, does the ACPI spec version matter? I assume yes. What decides the right version of ACPI spec in cloud-hypervisor? I guess the supported guest workload, e.g. the supported guest OS, etc?

Reply:
We simply rely on whatever is the latest specification at the moment. In this case the latest is https://uefi.org/sites/default/files/resources/ACPI_6_3_May16.pdf
Also, we always expect very recent guest kernels, which means they should have the support for all the features we're supporting through ACPI.